bin/cml/runner.js

```diff
@@ -553,6 +556,12 @@ exports.builder = (yargs) =>
         description: 'Specifies the subnet to use within AWS',
         alias: 'cloud-aws-subnet-id'
       },
+      cmlVersion: {
+        type: 'string',
+        default: require('../../package.json').version,
```

Comment:
:eyes: is it safe to assume this works? I doubt it, esp. for the binary distributions

Reply:
Yes, even for the binary distributions. Vercel [`pkg`](https://github.com/vercel/pkg) looks for calls to the `require` function and adapts them appropriately to the virtual file system.

Reply:
(Tested and confirmed)